x-pack/test/cloud_security_posture_functional/page_objects/csp_dashboard_page.ts

```diff
@@ -105,20 +105,29 @@ export function CspDashboardPageProvider({ getService, getPageObjects }: FtrProv
     getFindingsLinks: async (tab: (typeof TAB_TYPES)[keyof typeof TAB_TYPES]) => {
       await dashboard.getDashoard(tab);
       const pageContainer = await testSubjects.find('pageContainer');
-      return await pageContainer.findAllByXpath(`//button[contains(@class, 'euiLink')]`);
+      return [
+        await pageContainer.findByTestSubject('dashboard-summary-passed-findings'),
+        await pageContainer.findByTestSubject('dashboard-summary-failed-findings'),
+        ...(await pageContainer.findAllByTestSubject('grouped-findings-evaluation-link')),
+        ...(await pageContainer.findAllByTestSubject('view-all-failed-findings')),
+        ...(await pageContainer.findAllByTestSubject('benchmark-section-bench-name')),
+        ...(await pageContainer.findAllByTestSubject('benchmark-asset-type')),
+        ...(await pageContainer.findAllByTestSubject('compliance-score-section-passed')),
+        ...(await pageContainer.findAllByTestSubject('compliance-score-section-failed')),
+      ];
```

Comment:
Wouldn't this array be inconsistent in its values? the first two items will be arrays while the others will be spreaded elements. I think it would be better if we either spread all items or not spread any of them.